ui/pages/settings/developer-options-tab/developer-options-tab.tsx

```diff
@@ -44,6 +44,11 @@ const DeveloperOptionsTab = () => {
   const [hasResetOnboarding, setHasResetOnboarding] = useState(false);
   const [isServiceWorkerKeptAlive, setIsServiceWorkerKeptAlive] =
     useState(true);
+  const [enableNetworkRedesign, setEnableNetworkRedesign] = useState(
```

Comment:
Hmm I can't figure out why I don't see the `Settings -> Developer Options` mentioned in the PR testing steps.

<img width="547" alt="Screenshot 2024-06-10 at 4 31 10 PM" src="https://github.com/MetaMask/metamask-extension/assets/3500406/3316dd97-2b48-4f99-a11a-a2cba603b501">

Reply:
Nevermind, looks like an env var `ENABLE_SETTINGS_PAGE_DEV_OPTIONS=true`